### PR TITLE
Add intro to R and dplyr lesson

### PR DESCRIPTION
**Pull request by taylorreiter**
I'll be happy if I get through the first part of group_by and then skip to exports...tbd if this is too much material tho. I shoved a lot into collapsing sections to skim over while i teach but that ppl can reference if they revisit the lesson. I also just cut lots of stuff out, and then dumped some of that in the extras section at the bottom.

### CODE REVIEW

File: docs/arcadia-users-group/20231031-intro-to-r/lesson.md [hunk 1]
```diff
@@ -0,0 +1,1674 @@
...
+## # ℹ 4 more variables: genus <chr>, species <chr>, taxa <chr>, plot_type <chr>
+```
+
+We can also use multiple conditions in one `filter()` statement.
+Here we will get rows with a year less than or equal to 1988 and whose hindfoot length values are not `NA`.
+The `!` before the `is.na()` function means "not".
+
+
+```r
+filter(surveys, year <= 1988 & !is.na(hindfoot_length))
```
Comment (elizabethmcd) [inline docs/arcadia-users-group/20231031-intro-to-r/lesson.md:727]: Not sure if it comes up later, could be good to show `drop_na()` for getting rid of any rows where any column has an NA

File: docs/arcadia-users-group/20231031-intro-to-r/lesson.md [hunk 2]
```diff
@@ -0,0 +1,1674 @@
...
+##  9 DM         6        42.1
+## 10 DM         7        43.2
+## # ℹ 290 more rows
+```
+
+Data are often recorded in spreadsheets in a wider format, but lots of `tidyverse` tools, especially `ggplot2`, like data in a longer format, so `pivot_longer()` is often very useful.
+
+### Joining data together with `dplyr`
+
+Another operation that is often useful is to join two data.frames together based on shared columns.
```
Comment (elizabethmcd) [inline docs/arcadia-users-group/20231031-intro-to-r/lesson.md:1672]: I would maybe quickly demo this, can cut it out at the end if running out of time (you already have quite a bit of material!) but getting to this point would equip learners with most of what they need. Joins are usually what people need for a majority of things

**Review by elizabethmcd — APPROVED** [review]
LGTM! just two comments about maybe something to add or expand on.

File: docs/arcadia-users-group/20231031-intro-to-r/lesson.md (hunk 1 above)
Comment (taylorreiter) [reply docs/arcadia-users-group/20231031-intro-to-r/lesson.md:727]: 👍 will add it to the extras section at the end!

File: docs/arcadia-users-group/20231031-intro-to-r/lesson.md (hunk 2 above)
Comment (taylorreiter) [reply docs/arcadia-users-group/20231031-intro-to-r/lesson.md]: added!

**Pull request merged**